mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -316,6 +316,7 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_6100 setJumping (Z)V
 		ARG 1 jumping
 	METHOD method_6101 isClimbing ()Z
+	METHOD method_6102 isMobOrPlayer ()Z
```

Comment:
Name this by its usage instead of its implementation.

Reply:
Right now, this method is used to determine which living entities a vindicator will follow, as well as which mobs a wither will target. However, these are unlikely to remain the method's only uses in the future, and I think the name given here is more suitable than say, `isTargetableByVindicatorOrWither`.

Reply:
`isLivingTarget` maybe ?

Reply:
That could be confused with 'is this an instance of `LivingEntity`'.

Reply:
Turns out `isMobOrPlayer` is actually better (after looking at minecraft wiki)